lib/find.js

```diff
@@ -8,59 +8,89 @@ const fs = require('fs'),
 
 /**
  * Find all registered serial ports by-id
- * @param  {Function} callback
- * @return {Promise}
+ *
+ * @return {Promise<Object[]|Error>}
  */
 function find () {
-
+  const serialPath = this.paths.linux.serial;
```

Comment:
good idea. ` node-cmd ` may be a better option, it has over 350k users at the moment. looks like it gets the almost same number of installs per mpnth that one has for all time.

https://github.com/RIAEvangelist/node-cmd